examples/obs-csi-plugin/kubernetes/bucket/sc.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: obs-sc
+provisioner: obs.csi.huaweicloud.com
```

```suggestion
provisioner: obs.csi.huaweicloud.com
allowVolumeExpansion: true
```
the OBS-CSI supports capacity expansion, pls add this atribute

---

done